code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -60,16 +60,14 @@
 	set src in oview(1)
 
 	if(config.ghost_interaction)
-		src.dir = turn(src.dir, 90)
+		setDir(turn(dir, 90))
 		handle_rotation()
 		return
-	else
-		if(!usr || !isturf(usr.loc))
-			return
-		if(usr.stat || usr.restrained())
+
+		if(usr.incapacitated())

```

Comment:
This is all indented too far; this will only ever work if `config.ghost_interaction` is enabled.